lib/Collaboration/Reference/TalkReferenceProvider.php

```diff
@@ -287,4 +289,36 @@ protected function getRoomType(Room $room): string {
 				return 'unknown';
 		}
 	}
+
+	/**
+	 * @inheritDoc
+	 */
+	public function getId(): string {
+		return Application::APP_ID;
+	}
+
+	/**
+	 * @inheritDoc
+	 */
+	public function getTitle(): string {
+		return $this->l->t('Talk');
```

Comment:
Maybe something more explicit here, like "Talk rooms" or "Talk conversations". This is what's display in the provider list.

Reply:
Conversations then. Room is not used in the frontend